packages/teams-js/src/private/chat.ts

```diff
@@ -18,64 +18,47 @@ import { ChatMembersInformation } from './interfaces';
  */
 export namespace chat {
   /**
-   * @hidden
-   * Hide from docs
-   * --------------
-   * Allows the user to start or continue a conversation with each subentity inside the tab
    *
+   * Allows the user to start a direct chat with a user
```

Comment:
We should follow up with @erinha about what the right style and level of detail we are looking for in comments on exported functions. (It's also okay with me if you want to handle that when you go through and look at all of the comments and tests on the whole suite of deep linking issues)